src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/features/entrypoint/Entrypoint.tsx

```diff
@@ -33,6 +37,41 @@ export default function Entrypoint() {
     }
   };
 
+  const fetchExistingInstances = async () => {
+    try {
+      // const { data } = await get(getActiveInstancesUrl(selectedParty.partyId));'
+      // TODO: remove this dummy api response when api has been fixed
+      const data: ISimpleInstance[] = [
```

Comment:
would this be `data.instances` from the API?

Reply:
No, returns a plain array.